optimade/server/config.py

```diff
@@ -273,6 +273,17 @@ def set_implementation_version(cls, v):
         res.update(v)
         return res
 
+    @validator("root_path", pre=False)
+    def remove_end_slashes(cls, value: str) -> str:
+        """Remove ending slashes from root_path"""
+        if isinstance(value, str):
+            while value.endswith("/"):
+                value = value[:-1]
+            while value.startswith("/"):
+                value = value[1:]
```

Comment:
```suggestion
value = value.strip("/")
```
I think it would be better to use strip here as it is shorter.